plugin/cassandra/plugin.go

```diff
@@ -90,106 +90,151 @@ package main
 }
 `,
 		Defaults: `
 {
-  "host"     : "127.0.0.1",
-  "port"     : "9042",
-  "user"     : "cassandra",
-  "password" : "cassandra",
-  "datadir"  : "/var/vcap/store/cassandra/data"
+  "host"              : "cassandra_host", // "12.0.0.1"
+  "port"              : "9042", // "cassandra_port"
+  "user"              : "cassandra_user",     //cassandra
+  "password"          : "cassandra_password", // "cassandra"
+  "include_keyspaces" : [""],
+  "exclude_keyspaces" : [ "system_schema", "system_distributed", "system_auth", "system", "system_traces" ],
+  "save_users"        : true,
+  "bindir"            : "/var/vcap/jobs/cassandra/bin", 
+  "datadir"           : "/var/vcap/store/cassandra/data" 
 }
 `,
 		Fields: []plugin.Field{
-			plugin.Field{
+			{
 				Mode:     "target",
 				Name:     "host",
 				Type:     "string",
 				Title:    "Cassandra Host",
 				Help:     "The hostname or IP address of your Cassandra server.",
 				Default:  "127.0.0.1",
-				Required: true,
+				Required: false,
 			},
-			plugin.Field{
+			{
 				Mode:     "target",
 				Name:     "port",
 				Type:     "port",
 				Title:    "Cassandra Port",
 				Help:     "The 'native transport' TCP port that Cassandra server is bound to, listening for incoming connections.",
 				Default:  "9042",
-				Required: true,
+				Required: false,
 			},
-			plugin.Field{
+			{
 				Mode:     "target",
 				Name:     "user",
 				Type:     "string",
 				Title:    "Cassandra Username",
 				Help:     "Username to authenticate to Cassandra as.",
 				Default:  "cassandra",
-				Required: true,
+				Required: false,
 			},
-			plugin.Field{
+			{
 				Mode:     "target",
-				Name:     "password",
+				Name:     "cassandra",
 				Type:     "password",
 				Title:    "Cassandra Password",
-				Help:     "Password to authenticate to Cassandra as.",
+				Help:     "Password to auth_userenv",
 				Default:  "cassandra",
-				Required: true,
+				Required: false,
 			},
-			plugin.Field{
+			{
 				Mode:     "target",
-				Name:     "keyspace",
-				Type:     "string",
-				Title:    "Keyspace to Backup",
-				Help:     "The name of the keyspace to backup.",
-				Example:  "system",
-				Required: true,
+				Name:     "include_keyspaces",
+				Type:     "array",
+				Title:    "Keyspaces to Include in the Backup",
+				Help:     "The name of the keyspace to include in the backup.",
+				Example:  "['system']",
+				Required: false,
+			},
+			{
+				Mode:     "target",
+				Name:     "exclude_keyspaces",
+				Type:     "array",
+				Title:    "Keyspaces to Exclude from Backup",
+				Help:     "The name of the keyspace to exclude from backup.",
+				Example:  "[system']",
+				Default:  "['system_schema', 'system_distributed', 'system_auth', 'system', 'system_traces']",
+				Required: false,
+			},
+			{
+				Mode:     "target",
+				Name:     "save_users",
+				Type:     "boolean",
+				Title:    "to restore users and permissions along with the keyspaces that are restored",
```

Comment:
This will be rendered as the label on the v8 Web UI form for configuring this plugin, so it should probably be capitalized, and worded more appropriately.  Suggestion: "Also backup / restore users and permissions"